coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -483,6 +483,9 @@ def compute_segment_coding_progress(self, segment_id, messages=None):
         wrong_scheme_messages = 0
         not_coded_messages = 0
 
+        if len(messages) == 0:
+            return MessagesMetrics(len(messages), messages_with_labels, wrong_scheme_messages, not_coded_messages)
```

Comment:
Why do you need a short circuit return?

If you do need one, consider:
`return MessagesMetrics(0, 0, 0, 0)`
I think that's clearer in this case.